src/sections/layout/alert/Alert.tsx

```diff
@@ -0,0 +1,62 @@
+import { Alert as AlertBS } from 'react-bootstrap'
+import { ReactNode, useState } from 'react'
+import {
+  CheckCircleFill,
+  ExclamationCircleFill,
+  ExclamationTriangleFill,
+  InfoCircleFill
+} from 'react-bootstrap-icons'
+
+type AlertVariant = 'success' | 'info' | 'warning' | 'danger'
+
+interface AlertProps {
+  variant: AlertVariant
+  dismissible?: boolean
+  customHeading?: string
+  children: ReactNode
+}
+
+export function Alert({ variant, dismissible = true, customHeading, children }: AlertProps) {
+  interface AlertIcons {
+    [key: string]: JSX.Element
+  }
+  interface AlertHeadings {
+    [key: string]: string
+  }
+  const ALERT_ICONS: AlertIcons = {
+    success: <CheckCircleFill></CheckCircleFill>,
+    info: <InfoCircleFill></InfoCircleFill>,
+    warning: <ExclamationTriangleFill></ExclamationTriangleFill>,
+    danger: <ExclamationCircleFill></ExclamationCircleFill>
+  }
+
+  const ALERT_HEADINGS: AlertHeadings = {
+    success: 'Success!',
+    info: 'Information',
+    warning: 'Warning',
+    danger: 'Error'
+  }
+  const [show, setShow] = useState(true)
+
+  function getAlertIcon(variant: AlertVariant): JSX.Element {
+    return ALERT_ICONS[variant]
+  }
+  function getAlertHeading(variant: AlertVariant, customHeading?: string): string {
+    return customHeading ?? ALERT_HEADINGS[variant]
+  }
+  const heading = getAlertHeading(variant, customHeading)
+
+  return (
+    <>
+      {show && (
+        <AlertBS variant={variant} onClose={() => setShow(false)} dismissible={dismissible}>
+          <span role="img" aria-label={`alert-icon-${variant}`}>
+            {getAlertIcon(variant)}
+          </span>{' '}
+          &nbsp;
+          <b>{heading}</b> - {children}
+        </AlertBS>
+      )}
+    </>
+  )
+}
```

Comment:
I was thinking that maybe now it makes sense to move AlertIcon to its own component, so we can move all the alert icon stuff to another file. Because the Alert function got really huge with all the declarations at the beginning.

```
return (
     <>
       {show && (
         <AlertBS variant={variant} onClose={() => setShow(false)} dismissible={dismissible}>
           <AlertIcon variant={variant}/>
           &nbsp;
           <b>{heading}</b> - {children}
         </AlertBS>
       )}
     </>
   )
```

Reply:
good idea, I just checked in code with a separate AlertIcon component

Reply:
Regarding these related components, do you think we should copy the bootstrap style of using dot notation, so that only one import statement is needed?  For example here, with Alert.Link: https://react-bootstrap.github.io/components/alerts/